troposphere/themes/README.md

```diff
@@ -0,0 +1,73 @@
+# Troposphere Theme Guide
+_Note: Troposphere Theming is about to undergo a major change_
+
+Troposphere themes can change the app's colors, the logo on the app bar, and some images to suit your organization's identity.
+The theme is a CSS overwrite of the Bootstrap framework. The CSS is generated from the `.scss` SASS transpiled syntax. 
+
+- [SASS](http://sass-lang.com/)
+- [BootStrap](http://getbootstrap.com/)
+
+## Getting Started
+Because new themes are not part of the Troposphere repository, new themes are symlinked to `./troposphere/themes/` from somewhere outside of the root Troposphere directory.
+
+_Note: We recomend creating a git repository for your theme_
+
+#### Copy Theme
+
+Start by copying a theme from the Troposphere theme folder to a location outside the root Troposphere directory.
+
+From within the troposphere theme directory run:
+
+```cp -r troposphere_theme {path to your theme}/your-theme-name_theme```
```

Comment:
To get "code fencing" format, you need to have the three ticks on their own line:

```
cp -r troposphere_theme {path to your theme}/your-theme-name_theme
```